rust/operator-binary/src/controller.rs

```diff
@@ -62,10 +64,12 @@ use stackable_operator::{
         statefulset::StatefulSetConditionBuilder,
     },
     time::Duration,
+    utils::COMMON_BASH_TRAP_FUNCTIONS,
 };
 use strum::{EnumDiscriminants, IntoStaticStr};
 use tracing::Instrument;
 
+use crate::operations::graceful_shutdown::add_graceful_shutdown_config;
```

Comment:
Let's group this with the other crate-level imports.

Reply:
https://github.com/stackabletech/nifi-operator/pull/528/commits/7c63a164caaec382d733dc5d0b6a277a8559dc49